gateway/gateway_test.go

```diff
@@ -11,10 +11,14 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"net/url"
+	"os"
 	"reflect"
+	"strconv"
 	"testing"
 	"time"
 
+	kitHelper "github.com/rudderlabs/rudder-go-kit/testhelper"
```

Comment:
[minor] for consistency
```suggestion
	kithelper "github.com/rudderlabs/rudder-go-kit/testhelper"
```